### DISCUSSION TITLE
What's the purpose of "webServiceUrl" and "authenticationToken" inside pass.json?

### DISCUSSION THREAD

**Discussion by mbappai** [alexandercerutti/passkit-generator#132] [Q&A] [answered]
Hey Alex! I'd please like to know the purpose of these two fields in the pass.json file.

**Comment by alexandercerutti** [comment, marked as answer]
You can check them out here: https://developer.apple.com/documentation/walletpasses/pass

**Reply by mbappai** [reply]
Oh okay thank you! I'll check it out
